Source/Common/Debug.cpp

```diff
@@ -6,13 +6,15 @@
 #if ORCHESTRATOR_DEBUG
 #include <stdarg.h>
 #include <stdio.h>
-void DebugPrint(const char* format, ...)
+#include "dfprintf.h"
+void debug_print(const char* format, ...)
```

Comment:
You're right, I'm a fool. I'll change it back. I think I got confused because I was working on a different project with a different naming convention at the time :s. It'll change back as of `d5e3c34`.